serve.go

```diff
@@ -221,7 +221,11 @@ func serve(args []string) error {
 	}
 
 	if *flRepoPath != "" {
-		r.PathPrefix("/repo/").Handler(http.StripPrefix("/repo/", http.FileServer(http.Dir(*flRepoPath))))
+		if _, err := os.Stat(*flRepoPath); err == nil {
```

Comment:
Use log.Fatal early, and drop the `else`. In Go it's generally better to avoid `else` in favor of early returns.